README.md

```diff
@@ -1,369 +1,168 @@
-HOME_GAS_PRICE | Gas Price to use for transactions to relay withdraws to Home Network
-FOREIGN_RPC_URL | Public RPC Node URL for Foreign Network  
-FOREIGN_OWNER_MULTISIG | Address of Administrator role on FOREIGN network to change parameters of the bridge and validator's contract
-FOREIGN_UPGRADEABLE_ADMIN_VALIDATORS | Address from which Validator's contract could be upgraded
-FOREIGN_UPGRADEABLE_ADMIN_BRIDGE | Address from which HomeBridge's contract could be upgraded
-FOREIGN_DAILY_LIMIT | Daily Limit in Wei. Example above is `1 eth`  
-FOREIGN_MAX_AMOUNT_PER_TX | Max limit per 1 tx in Wei. Example above is `0.1 eth`  
-FOREIGN_MIN_AMOUNT_PER_TX | Minimum amount per 1 tx in Wei. Example above is `0.01 eth`  
-FOREIGN_REQUIRED_BLOCK_CONFIRMATIONS | Number of blocks issued after the block with the corresponding withdraw transaction to make sure that the transaction will not be rolled back
-FOREIGN_GAS_PRICE | Gas Price to use for transactions to deposit and confirm withdraws to Foreign Network
-VALIDATORS | array of validators on Home and Foreign network. Space separated.  
-REQUIRED_NUMBER_OF_VALIDATORS | Minimum Number of validators in order to Withdraw Funds on POA network Sokol  
-
-3. run `node deploy.js`  
-
-As a result you should have the following output:  
-```bash
-[   Home  ] HomeBridge:  0xE460DD303Abf282Bde9f7c9cB608D1E1a7c02E0a
-[ Foreign ] ForeignBridge:  0x902a15b45a3cD1A8aC5ab97c69C8215FC26763eA
-[ Foreign ] POA20:  0xAb121C134aD7e128BE06fEaf40b494F9865F794b
-Contracts Deployment have been saved to `bridgeDeploymentResults.json`
-```
-DESTROY `.env` file  
-`rm -rf .env`  
-The deployment information is also located in `bridgeDeploymentResults.json`  
-When you are done, go back to `sokol-kovan-bridge` folder   
-`cd ../../`
+4. Fund the test account(s).
+  * Fund Home accounts (`Validators`) using the [POA Sokol Faucet](https://faucet-sokol.herokuapp.com/)
+  * Get free Kovan Coins from the [gitter channel](https://gitter.im/kovan-testnet/faucet) or [Iracus faucet](https://github.com/kovan-testnet/faucet) for Foreign Accounts. Get 5 Keth to 1 acc, and transfer from it to all other wallets.
 
-## 5. Install poa-bridge binary
 
-* Install `rust` and `cargo`: [installation instructions.](https://www.rust-lang.org/en-US/install.html)
+5. Deploy the Sokol <-> Kovan Bridge contracts.
+  * Go to the the `sokol-kovan-bridge` folder created in step 1 and `git clone https://github.com/poanetwork/poa-bridge-contracts`
+  * Follow instructions in the [POA Bridge contracts repo](https://github.com/poanetwork/poa-bridge-contracts).
+  * Set the parameters in the .env file.
+    * DEPLOYMENT_ACCOUNT_PRIVATE_KEY: Private key from step 2
+    * HOME_RPC_URL=https://sokol.poa.network
+    * HOME_OWNER_MULTISIG: Wallet address(es) of validators. For testing, you can use the same address for all address values in the file. 
```

Comment:
Actually the same is applicable for `HOME_OWNER_MULTISIG`, `HOME_UPGRADEABLE_ADMIN_VALIDATORS`, `HOME_UPGRADEABLE_ADMIN_BRIDGE`, `FOREIGN_OWNER_MULTISIG`, `FOREIGN_UPGRADEABLE_ADMIN_VALIDATORS` and `FOREIGN_UPGRADEABLE_ADMIN_BRIDGE`